src/Server/Helper.php

```diff
@@ -79,17 +81,17 @@ public function parseHttpRequest(?callable $readRawBodyFn = null)
             }
 
             if (stripos($contentType, 'application/graphql') !== false) {
-                $rawBody    = $readRawBodyFn
-                    ? $readRawBodyFn()
-                    : $this->readRawBody();
+                $rawBody    = $readRawBodyFn === null
+                    ? $this->readRawBody()
+                    : $readRawBodyFn();
                 $bodyParams = ['query' => $rawBody ?? ''];
             } elseif (stripos($contentType, 'application/json') !== false) {
-                $rawBody    = $readRawBodyFn ?
-                    $readRawBodyFn()
-                    : $this->readRawBody();
+                $rawBody    = $readRawBodyFn === null
+                    ? $this->readRawBody()
```

Comment:
In order to test these lines we would have to modify PHP globals and mock `php://input`. I think it's not worth and out of scope here.